executor_manager/executors/docker/executor.py

```
@@ -327,6 +327,90 @@ def delete_executor(self, executor_name: str) -> Dict[str, Any]:
+            cancel_url = f"http://localhost:{port}/api/tasks/cancel?task_id={task_id}"
+            logger.info(f"Calling cancel API for task {task_id} at {cancel_url}")
+
+            try:
+                response = self.requests.post(cancel_url, timeout=10)
+                response.raise_for_status()
+
+                logger.info(f"Successfully cancelled task {task_id}")
+                return {
+                    "status": "success",
+                    "message": f"Task {task_id} cancellation requested successfully"
+                }
```

Ruff (0.14.5)

Consider moving this statement to an `else` block

(TRY300)